itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolCatalogOps.java

```diff
@@ -35,7 +35,7 @@
 import org.apache.hadoop.hive.metastore.client.builder.PartitionBuilder;
 import org.apache.hadoop.hive.metastore.client.builder.TableBuilder;
 import org.apache.hadoop.hive.metastore.conf.MetastoreConf;
-import org.apache.hive.com.google.common.io.Files;
+import com.google.common.io.Files;
```

Comment:
nit: After the removal of shading pattern, this does not follow alphabetical order of imports

Reply:
Done.